index.js

```diff
@@ -52,6 +53,7 @@ module.exports = function(config) {
     (!config.cloudwatchNamespace && config.cloudwatchStackname)) throw new Error('both cloudwatchNamespace and cloudwatchStackname must be configured');
 
   config.maxProcessTime = config.maxProcessTime || 3e5;
+  config.minProcessTime = config.minProcessTime || 0;
```

Comment:
Considering how low the default will be (500ms-1000ms), the worst case scenario is we get slightly bigger batches. This assumes the `minProcessingTime` wouldn't be updated to be lower than the stack's minimum. I see this as a really basic protection against reading as fast as possible. If stacks want to wrap this in more complex logic, that's fine too.